### ISSUE TITLE
datasets

### ISSUE THREAD

**Issue by abcfjy** [zjukg/Enrich-on-Graph#4]
<img width="998" height="214" alt="Image" src="https://github.com/user-attachments/assets/9138d67f-95f0-469d-a9d4-e0e4de14830a" />

<img width="183" height="621" alt="Image" src="https://github.com/user-attachments/assets/c42e662b-fa14-41d4-adc2-c81400881365" />
"Could you please clarify how to configure the dataset? Do I need to distinguish between the training set and the test set? I would appreciate it if you could provide more detailed instructions. Looking forward to your reply."

**Comment by returnlsz** [comment]
Thank you for your interest in our work! You only need to work on the test set. You just need to merge the three test parquet files shown in your image into one parqurt file and fill in the path in DATASET_PATH. Additionally, we have already updated the enriched subgraphs on ModelScope (i.e., parsing, pruning, and enriching processes), and you can directly use these subgraphs for inference :D. Links: [CWQ](https://modelscope.cn/datasets/AGsz666/EoG-enrich-data-cwq) and [WebQSP](https://modelscope.cn/datasets/AGsz666/EoG-enrich-data-webqsp).
